share/spice/skyscanner_flight_search/footer.handlebars

```diff
@@ -0,0 +1,6 @@
+<div class="pull-left" style="font-size:1.2em">
```

Comment:
Let's use one of our `tx--##` font-size classes from https://ddg.gg/styleguide

e.g. `tx--21`